src/data_base.cpp

```diff
@@ -1 +1,94 @@
-ECHO is on.
+#include <jsoncpp/json/json.h>
+#include <iostream>
+#include <fstream>
+
+#include "data_base.hpp"
+#include "string_utils.hpp"
+namespace genetics {
+
+namespace {
+	std::pair<std::string, std::string> inputValidator(const std::string& a_geneName, const std::string& a_variant)
```

Comment:
maybe decler as using (the pair) it in the database.hpp and use it there too instead of writting twice